scripts/integration_shards/main.go

```
@@ -0,0 +1,751 @@
+		}
+		switch event.Action {
+		case "run":
+			started[event.Test]++
+		case "pass", "skip", "fail":
+			terminals[event.Test] = append(terminals[event.Test], event.Action)
+		case "output":
+			outputs[event.Test] = append(outputs[event.Test], event.Output)
+		}
+	}
+	if eventCount == 0 {
+		verificationErrors = append(verificationErrors, errors.New("Go test event stream contained zero events"))
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)